.github/workflows/pull-request.yml

```diff
@@ -58,8 +58,9 @@ jobs:
         uses: nick-fields/retry@v2
         with:
           max_attempts: 3
-          timeout_minutes: 20
-          command: npm run test:e2e -- -- --log-timestamp --install-wrangler=3.8.0
+          timeout_minutes: 10
+          # Note: the latest wrangler version compatible with pages-e2e is 3.17.0
+          command: npm run test:e2e -- -- --log-timestamp --install-wrangler=3.17.0
```

Comment:
Are newer versions of wrangler not compatible? How?

Reply:
It seems like the issue is caused by the pages-e2e testing framework using `--port=0`: https://github.com/cloudflare/workers-sdk/issues/4563